typed/typed.go

```diff
@@ -110,6 +111,18 @@ func (c *Comparison) IsSame() bool {
 	return c.Removed.Empty() && c.Modified.Empty() && c.Added.Empty()
 }
 
+// String returns a human readable version of the comparison.
+func (c *Comparison) String() string {
```

Comment:
Maybe call this function from the command line tool if you're already changing things?